docs/website/docs/installation.mdx

```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
 
 ## Set up environment
 
-## Install `dlt` library
\ No newline at end of file
+Make sure you are using **Python 3.8 or 3.9**:
```

Comment:
everything > 3.8 should work. the bigquery client does not work on python 3.11 yet but dlt itself runs on it